### ISSUE TITLE
Second Question

### ISSUE THREAD

**Issue by linnabrown** [cambridge-mlg/cnaps#4]
https://github.com/cambridge-mlg/cnaps/blob/5a11dc0c450b0b0728de98eb64a0d1bc83035a2e/src/model.py#L108-L110

Why don't you utilize these params `train_features` and `train_labels` in your function `_get_classifier_params` ?

**Comment by jfb54** [comment]
Thanks for pointing this out. At some point during development of the code, we refactored the functionality  of `_get_classifier_params(self, train_features, train_labels)` into two methods (a combination of `_build_class_reps(self, context_features, context_labels)` and `_get_classifier_params`) and neglected to realize that `_get_classifier_params` no longer needed any arguments as if uses the class member variable `self.class_representations` instead. There is no bug here, just a bit of sloppy refactoring. Note that `train_features` is the same as `context_features` and `train_labels` is the same as `context_labels`. At some point we changed the names in the code to align with those in the paper and missed these.
